README.md

```diff
@@ -154,6 +155,27 @@ app.service('users').find({
 GET /users?name[$like]=Dav%
 ```
 
+## Modifying the Model
+
+Sequelize allows you to call methods like `Model.scope()`, `Model.schema()`, and others. To use these methods, extend the class to overwrite the `getModel` method.
+
+```js
+const { SequelizeService } = require('feathers-sequelize');
+
+class Service extends SequelizeService {
+  getModel(params) {
+    let Model = this.options.Model;
+    if (params?.sequelize?.scope) {
+      Model = Model.scope(params?.sequelize?.scope);
```

Comment:
You could remove the optional chaining (`?.`). I think, if you use the chains, ts complains that you can't pass `undefined` to `Model.scope()` even though it should know that the optional chaining is useless.

It's just docs and everyone should be capable to fix this for themselves, but yeah.